src/components/ApplySuccessModal.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <PopupModal v-if="modelValue" @close="close">
+
+    <template #header>
+      <h2>申請成功</h2>
+    </template>
+
+    <template #content>
+      <h3>櫃子編號：{{ locker.name }}</h3>
+      <h3>借用類型：{{ borrowType }}</h3>
+      <h3>借用時間：{{ timeRange.start }} ~ {{ timeRange.end }}</h3>
+      <h3>借用理由：
+        <div class="borrow-reason">
+        <p>{{ reason }}</p>
```

Comment:
```suggestion
          <p>{{ reason }}</p>
```
這邊好像少按一下tab，少了兩個space。沒問題的話直接按Apply Suggestion即可！